code/modules/mob/new_player/login.dm

```diff
@@ -27,7 +27,7 @@
 		loc = pick(watch_locations)
 */
 
-	callHook("mob_login", list("client" = client, "mob" = src))
+	update_morgue() // Why is this called for new players that dont even have a morgue
```

Comment:
Test if this can be removed, and, if so, do so, please.

Reply:
Will do